### ISSUE TITLE
Error while updating [Call to undefined method ...]

### ISSUE THREAD

**Issue by dartcafe** [nextcloud/polls#3057]
### Annoncement

This error is raised, if methods of the migration or repair steps get changed. The issue is handled here: https://github.com/nextcloud/server/issues/38158

Unfortunately we can do nothing in Polls, except stop changing this methods. 

**Edit:** Just repeat the update, the classes are loaded correctly then.

**Comment by solracsf** [comment]
Yep...
```
polls new version available: 5.3.0
An unhandled exception has been thrown:
Error: Call to undefined method OCA\\Polls\\Db\\TableManager::setConnection() in /apps/polls/lib/Migration/RepairSteps/RemoveObsoleteMigrations.php:56
Stack trace:
#0 /lib/private/Repair.php(126): OCA\\Polls\\Migration\\RepairSteps\\RemoveObsoleteMigrations->run()
#1 /lib/private/legacy/OC_App.php(1077): OC\\Repair->run()
#2 /lib/private/legacy/OC_App.php(1014): OC_App::executeRepairSteps()
#3 /lib/private/Installer.php(200): OC_App::updateApp()
#4 /core/Command/App/Update.php(107): OC\\Installer->updateAppstoreApp()
#5 /3rdparty/symfony/console/Command/Command.php(255): OC\\Core\\Command\\App\\Update->execute()
#6 /3rdparty/symfony/console/Application.php(1009): Symfony\\Component\\Console\\Command\\Command->run()
#7 /3rdparty/symfony/console/Application.php(273): Symfony\\Component\\Console\\Application->doRunCommand()
#8 /3rdparty/symfony/console/Application.php(149): Symfony\\Component\\Console\\Application->doRun()
#9 /lib/private/Console/Application.php(211): Symfony\\Component\\Console\\Application->run()
#10 /console.php(100): OC\\Console\\Application->run()
#11 /occ(11): require_once('...')
#12 {main}
```

**Comment by dartcafe** [comment]
Since the problem did not raise anymore either with 27.1.3 nor with 28-beta1, I think we can unpin it.
